mapper/orm-coordination-outbox-polling/src/main/java/org/hibernate/search/mapper/orm/coordination/outboxpolling/cluster/impl/OutboxPollingAgentAdditionalJaxbMappingProducer.java

```diff
@@ -129,30 +136,33 @@ public Collection<MappingDocument> produceMappings(ConfigurationPropertySource p
 		Optional<String> schema = ENTITY_MAPPING_AGENT_SCHEMA.get( propertySource );
 		Optional<String> catalog = ENTITY_MAPPING_AGENT_CATALOG.get( propertySource );
 		Optional<String> table = ENTITY_MAPPING_AGENT_TABLE.get( propertySource );
-		Optional<String> generator = ENTITY_MAPPING_AGENT_GENERATOR.get( propertySource );
+		Optional<UuidGenerationStrategy> uuidStrategy = ENTITY_MAPPING_AGENT_UUID_GEN_STRATEGY.get( propertySource );
+
 
 		// only allow configuring the entire mapping or table/catalog/schema/generator names
-		if ( mapping.isPresent() && ( schema.isPresent() || catalog.isPresent() || table.isPresent() || generator.isPresent() ) ) {
+		if ( mapping.isPresent() && ( schema.isPresent() || catalog.isPresent() || table.isPresent() ) ) {
```

Comment:
This was supposed to check for the UUID generation strategy and UUID data type too, wasn't it?

I suspect the same problem exists for outbox events.

Reply:
hmm I remember thinking about this the previous year, and there was some reason why I thought it was not necessary, but now I cannot remember what it was, will make the adjustments for the outbox event too.
(and there - a joke about the prev year can be crossed out from the list 😄 )